config/packages/framework.yaml

```diff
@@ -3,7 +3,7 @@ framework:
     trusted_proxies: '%env(TRUSTED_PROXIES)%'
     trusted_headers: ['x-forwarded-for', 'x-forwarded-host', 'x-forwarded-proto', 'x-forwarded-port']
     secret: '%env(APP_SECRET)%'
-    #csrf_protection: true
+    csrf_protection: true
```

Comment:
In principe de default dus wijziging was niet nodig geweest, maar ach, zo is het duidelijker :)